timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,151 @@
-# 5. Query your data
+# Query your data
 
 With TimescaleDB, there's no need to learn a custom query language. **TimescaleDB
-supports full SQL**. This means you can put your SQL knowledge to good use and
-use the rich ecosystem of PostgreSQL tools you know and love.
-
-For example, here's how to find the average temperature for each city in the past 2 years:
-
-```sql
---------------------------------
--- Average temperature per city
--- in past 2 years
---------------------------------
-SELECT city_name, avg(temp_c)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '2 years'
-GROUP BY city_name;
-```
-
-And here's how to find the total snowfall for each city in the past 5 years:
-
-```sql
---------------------------------
--- Total snowfall per city
--- in past 5 years
---------------------------------
-SELECT city_name, sum(snow_1h_mm)
-FROM weather_metrics
-WHERE time > now() - INTERVAL '5 years'
-GROUP BY city_name;
-```
-
+supports full SQL**. You can use your SQL knowledge along with the rich 
+ecosystem of PostgreSQL tools in addition to the extra functionality gained through
+TimescaleDB.
 
 <highlight type="tip">
 Fun fact: TimescaleDB adds important enhancements to the PostgreSQL query planner
 that improve query reusability for INTERVAL predicates, something PostgreSQL does
 not have.
 </highlight>
 
-## Advanced SQL functions for time-series data
+Here we provide some query examples so that you can get familiar with using SQL along
+with some of TimescaleDB's most popular functions. 
+
+<procedure>
```

Comment:
I don't think this really qualifies as a procedure, since it's actually a list of commands to try (not steps that you need to work through in order). Might be better to take it out of procedure tags, and just do it as a bulleted list?

Reply:
Yeah I actually made that exact change yesterday evening and forgot to push it up! Thanks Lana!